### PR TITLE
#159250132 Fix python coding errors flake8

### PR DESCRIPTION
**Pull request by magicmarie**
## What does this PR do?
Fixes python coding errors from flake8

## Description of Task to be completed?
Correct any errors related to ```long lines```, ```imports not at the top of the file```, ```indentation``` among others.

## How should this be manually tested?
After cloning the repo, cd into it and run ```flake8```

Any background context you want to provide?
Helps prevent things like syntax errors, typos, bad formatting, incorrect styling etc

What are the relevant pivotal tracker stories?
[#159250132](https://www.pivotaltracker.com/story/show/159250132)

### CODE REVIEW

**Comment by isheebo** [comment]
@magicmarie kindly squash all your commits into one

**Comment by matthewacha** [comment]
Lint settings.py as well

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-web: gunicorn wger.wsgi.application
+web: gunicorn wger.wsgi:application
```
Comment (isheebo) [inline Procfile]: I don't think this should be a part of this PR

File: package-lock.json [hunk 2]
```diff
@@ -257,7 +257,12 @@
     "bower": {
       "version": "1.8.4",
       "resolved": "https://registry.npmjs.org/bower/-/bower-1.8.4.tgz",
+<<<<<<< HEAD
```
Comment (isheebo) [inline package-lock.json]: This file should also not be a part of your PR

File: requirements.txt [hunk 3]
```diff
@@ -1,16 +1,27 @@
 alabaster==0.7.11
+<<<<<<< HEAD
```
Comment (isheebo) [inline requirements.txt]: This file too shouldn't be part of the PR

File: settings.py [hunk 4]
```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import dj_database_url
+# import dj_database_url
```
Comment (isheebo) [inline settings.py]: This import is actually needed in the project. Please 'uncomment' it

File: settings.py [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
 from decouple import config
-from wger.settings_global import *
+import os
+from wger.settings_global import *  # noqa E403
```
Comment (isheebo) [inline settings.py]: sort these imports, the ones from the standard library, followed by the ones from third party and then imports from your application

File: tasks.py [hunk 6]
```diff
@@ -180,7 +199,8 @@ def create_settings(context, settings_path=None, database_path=None, url=None, d
         settings_file.write(settings_content)
 
 
-@task(help={'settings-path': 'Path to settings file (absolute path recommended). Leave empty for default'})
+@task(help={'settings-path': 'Path to settings file (absolute path recommended).\
```
Comment (isheebo) [inline tasks.py]: format this such that each key rests on a new line

**Comment by magicmarie** [comment]
@isheebo changes made as requested

**Comment by mutebironald** [comment]
@magicmarie , could you have a single commit

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (isheebo) [inline .vscode/settings.json]: this file should not be pushed. It is your local vs code environment. Since anyone can create their own `.vscode/`, it is advisable not to push it

**Comment by isheebo** [comment]
you still have changes in files that are not lint-able. changes to `package-lock.json`, `release.sh`, and the `Procfile` shouldn't be in your PR

**Comment by isheebo** [comment]
LGTM
